### ISSUE TITLE
Just looking for a forum

### ISSUE THREAD

**Issue by tommymsw1** [OpenELEQ/verified-metalliq-players#12]
I can't find help anywhere for MetalliQ. Is there a forum? It does not work at all for me. Will not find ANY players.

**Comment by OpenELEQ** [comment]
?? So the issue is you can't find the support thread??
You must not have looked very hard as I have it linked in the addon.xml as well as in my twitter account description and on almost all of my yt-vids... 
In the future, please only submit actual issues relevant to the repo, thank you.

**Comment by tommymsw1** [comment]
Thank you for the reply, but you might have just added the support forum in your response. I guess I will Google "OpenELEC Twitter"? I Googled MetalliQ forum and I found a few forums but they are all way old and have not been used in some time. "linked in the addon.xml" does not mean anything to me and "all of my yt-vids" is also not English. Your answer implies I understand the references, but if I did, I would already know where to find the answer. :) 

And I did have a problem... (As I mentioned) MetalliQ does not see ANY of my video addons (but ONLY on my Android setup. It works fine on my PC setup with the same settings and addons)
